doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -187,8 +217,14 @@ end to end tests.**
 
 ##### Removing a deprecated feature
 
-- Announce deprecation and support policy of the existing feature
-- Deprecate the feature
+- We are adding a new --version flag to allow users to switch between
+v1alpha1 and the proposed v1alpha2 or vice-versa for backward compatiblity
```

Comment:
How long will keep v1alpha1?

Reply:
maybe only keep it for the next release/fixes (0.12->0.12.x), then remove it following any release after that?

Reply:
@dmesser @joelanford wdyt?

Reply:
Yeah I think keeping it for 1 release would be enough.

So, for example:
- 0.11.x has v1alpha1 only
- 0.12.x has v1alpha1 and v1alpha2
- 0.13.x has v1alpha2 only